python/GafferDispatch/LocalDispatcher.py

```diff
@@ -283,7 +283,10 @@ def __doBackgroundDispatch( self, batch ) :
 			self.__setStatus( batch, LocalDispatcher.Job.Status.Running )
 			IECore.msg( IECore.MessageHandler.Level.Info, self.__messageTitle, " ".join( args ) )
 			if os.name == "nt":
-				process = subprocess.Popen( args )
+				if self.__environmentCommand :
+					process = subprocess.Popen( subprocess.list2cmdline( args ), shell=True )
```

Comment:
Yes, I guess that is duplicating with that `Popen` would do anyways. I had included it based on a comment in the docs

> The shell argument (which defaults to False) specifies whether to use the shell as the program to execute. If shell is True, it is recommended to pass args as a string rather than as a sequence.

But I'll remove it since `Popen` will the same thing for us.

Reply:
Removed in https://github.com/GafferHQ/gaffer/pull/5036/commits/de8814bdbf0463205637667e296d5ec70f29204d